src/create_disk.sh

```diff
@@ -68,23 +68,12 @@ udevtrig() {
 export PATH=$PATH:/sbin:/usr/sbin
 arch="$(uname -m)"
 
-if [ -n "$platforms_json" ]; then
-    platform_grub_cmds=$(jq -r ".${arch}.${platform}.grub_commands // [] | join(\"\\\\n\")" < "${platforms_json}")
-    platform_kargs=$(jq -r ".${arch}.${platform}.kernel_arguments // [] | join(\" \")" < "${platforms_json}")
-else
-    # Add legacy kargs and console settings
-    platform_grub_cmds='serial --speed=115200\nterminal_input serial console\nterminal_output serial console'
-    DEFAULT_TERMINAL=$(. $(dirname "$0")/cmdlib.sh; echo $DEFAULT_TERMINAL)
-    # On each s390x hypervisor, a tty would be automatically detected by the
-    # kernel and systemd, there is no need to specify one.  However, we keep
-    # DEFAULT_TERMINAL as ttysclp0, which is helpful for building/testing
-    # with KVM+virtio (cmd-run).  For aarch64, ttyAMA0 is used as the
-    # default console
-    case "$arch" in
-        "aarch64"|"s390x") platform_kargs= ;;
-        *) platform_kargs="console=tty0 console=${DEFAULT_TERMINAL},115200n8" ;;
-    esac
+if [ -z "$platforms_json" ]; then
+    echo "Missing --platforms-json" >&2
+    exit 1
 fi
+platform_grub_cmds=$(jq -r ".${arch}.${platform}.grub_commands // [] | join(\"\\\\n\")" < "${platforms_json}")
```

Comment:
(This is one of those things where bash is really painful...the quotes and backslashes look like line noise to me.  )